qstash/api/api-ratelimiting.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "API Rate Limit Response"
+description: "This page documents the rate limiting behavior of our API and explains how to handle different types of rate limit errors."
+---
+
+## Overview
+
+Our API implements rate limiting to ensure fair usage and maintain service stability. There are three types of rate limits:
+
+1. **Daily rate limit**
+2. **Burst rate limit**
+3. **Chat-based rate limit**
+
+When a rate limit is exceeded, the API returns a 429 status code along with specific headers that provide information about the limit, remaining requests/tokens, and reset time.
+
+### Daily Rate Limit
+
+This is a **daily** limit applied across all API endpoints.
```

Comment:
The daily rate limit refers to the maximum number of messages that can be published per day. It is consumed by new message requests like publish, enqueue, or batch. Other API requests, such as fetching events or messages, do not count toward this limit.